.gitignore

```diff
@@ -1,3 +1,5 @@
 *.pyc
 .env
 .idea
+app_init_backUp.py
+database_config.py
```

Comment:
missing new line.

Generally you don't want to ignore every possible file you might have in your local environment but just files that are generated by running the project or configuration files that contain secrets.

For this kind you might want to ignore them or just delete them.